src/stories/Library/Lists/list-description/list-description.scss

```diff
@@ -27,6 +27,11 @@ dl.list-description {
     flex-direction: column;
   }
 
+  .list-description__value--list {
+    list-style: initial;
+    padding-left: 1em;
```

Comment:
Maybe I would like to see some of our pre-defined spacing variables here instead, as we use those all around (check out the variables.spacing file).

Reply:
Thank you, @Adamik10. I have updated it to`padding-left: $s-md;`. This translates to `16px` (where `1em` is approximately `12px`), and I actually think it looks better now. 👍🏻

<img width="375" alt="image" src="https://github.com/danskernesdigitalebibliotek/dpl-design-system/assets/105956/f0b8f1a0-86d1-40e7-a159-f451ff829723">